projects/storefrontapp-e2e-cypress/cypress.json

```diff
@@ -6,6 +6,6 @@
   "env": {
     "CLIENT_ID": "mobile_android",
     "CLIENT_SECRET": "secret",
-    "API_URL": "https://dev-com-17.accdemo.b2c.ydev.hybris.com:9002"
+    "API_URL": "https://dev-com-14.accdemo.b2c.ydev.hybris.com:9002"
```

Comment:
I know this is not the final version, I'm just leaving this comment so we don't forget to revert this file